### PR TITLE
Hybrid

### LINKED ISSUE #17: RMSE evaluation for hybrid on each parameter
Evaluate hybrid on each parameter, apart from floor, since this estimation is the same as scene analysis alone.

Evaluate hybrid on X- and Y-coordinates using Root Mean Squared Error.

### PR DESCRIPTION
**Pull request by MrPekar98**

### CODE REVIEW

**Review by MikkelFilip — APPROVED** [review]

File: hybrid/ml/ml_wrapper.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    # ANN next position estimation.
+    def __ann_next_position(self, data):
+        return self.algorithm.next_position()
+
+    # RNN next position estimation.
+    def __rnn_next_position(self, data):
+        pass
+
+    # KNN next position estimation.
+    def __knn_next_position(self, data):
```
Comment (ceciliehyrup1) [inline hybrid/ml/ml_wrapper.py:41]: her

**Review by melanieselman — APPROVED** [review]

File: hybrid/ml/ml_wrapper.py (hunk 1 above)
Comment (MrPekar98) [reply hybrid/ml/ml_wrapper.py:41]: I'm gonna pretend this never appeared.

**Review by ceciliehyrup1 — APPROVED** [review]

File: hybrid/ml/ml_wrapper.py (hunk 1 above)
Comment (ceciliehyrup1) [reply hybrid/ml/ml_wrapper.py:41]: Same (☞ﾟヮﾟ)☞☜(ﾟヮﾟ☜)

**Review by elaurs — APPROVED** [review]

**Comment by elaurs** [comment]
Hvordan unapprover man?

**Review by Abiram98 — APPROVED** [review]

**Pull request merged**